InfernalRobotics/InfernalRobotics/API/IRWrapper.cs

```diff
@@ -104,66 +101,74 @@ public static Boolean InitWrapper()
 
             try
             {
-                var fi = IRServoControllerType.GetProperty("Instance", BindingFlags.Public | BindingFlags.Static);
-                
-                if (fi == null)
+                var propertyInfo = IRServoControllerType.GetProperty("Instance", BindingFlags.Public | BindingFlags.Static);
+
+                if (propertyInfo == null)
                     LogFormatted("[IR Wrapper] Cannot find Instance Property");
-                actualServoController = fi.GetValue(null, null);
+                else
+                    ActualServoController = propertyInfo.GetValue(null, null);
             }
             catch (Exception e)
             {
                 LogFormatted("No Instance found, " + e.Message);
             }
 
-            if (actualServoController == null)
+            if (ActualServoController == null)
             {
                 LogFormatted("Failed grabbing Instance");
                 return false;
             }
 
             LogFormatted("Got Instance, Creating Wrapper Objects");
-            IRController = new IRAPI(actualServoController);
+            IRController = new InfernalRoboticsAPI(ActualServoController);
```

Comment:
This would be hell to use.

Imagine a call like this from Sequencer ``` foreach (IRWrapper.IRAPI.IRControlGroup g in IRWrapper.IRController.ServoGroups) ```